pom.xml

```diff
@@ -132,6 +132,8 @@
                     <descriptorRefs>
                         <descriptorRef>jar-with-dependencies</descriptorRef>
                     </descriptorRefs>
+                    <appendAssemblyId>false</appendAssemblyId>
+                    <finalName>hadoop-router-fs-hadoop-${hadoop.version}-assembly-${project.version}</finalName>
```

Comment:
This does not fall under the title of the PR.  Could you please rephrase the title or just move these lines to another PR?